iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -153,7 +151,6 @@ public double getTargetDeadband() {
 
     public C setTargetDeadband(double targetDeadband) {
         int variantIndex = network.get().getVariantIndex();
-        ValidationUtil.checkTargetDeadband(parent, type, this.regulating.get(variantIndex), targetDeadband);
```

Comment:
You can keep this one I think

Reply:
Now the method `checkTargetDeadband `is different for `ratioTapChanger `and `phaseTapChanger `as for `ratioTapChanger `the attribute `loadTapChangingCapabilities `has been added to be consistent with the rest of the `ratioTapChanger `methods.